doc/example/org.obeonetwork.m2doc.ecore.wtable.test/refreshDiag/README.txt

```diff
@@ -0,0 +1,14 @@
+To execute refresh test : 
+1 - generate with no model modification with �testRefreshDiagTagToFalse.genconf� or �testRefreshDiagTagToTrue.genconf� launcher

```

Comment:
There are unknown characters in the text file?!?
Please use standard single or double quotes.